cpp/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ project(arrow VERSION "${ARROW_BASE_VERSION}")
 set(ARROW_VERSION_MAJOR "${arrow_VERSION_MAJOR}")
 set(ARROW_VERSION_MINOR "${arrow_VERSION_MINOR}")
 set(ARROW_VERSION_PATCH "${arrow_VERSION_PATCH}")
-if (ARROW_VERSION_MAJOR STREQUAL "" OR
+if(ARROW_VERSION_MAJOR STREQUAL "" OR
```

Comment:
I'm not fond of the style changes here. If we want to reformat the cmake files we should do so consistently in a separate PR, IMHO. Here it's making it more tedious to follow your changes.

(also it seems there are automatic tools for that, such as https://github.com/cheshirekow/cmake_format)

Reply:
O.K, Let's discuss the style stuff in another PR.